_info/Tribute

```diff
@@ -1,6 +1,6 @@
 Tribute
 
-Every day phase, the king has to choose to tribute one of this team members. This is mandatory. The king has to also choose a certain amount of money to assign to the person. Each team will do the same. 
+Every day phase, each team's king has to choose to tribute one of this team members. This is mandatory. They have to also choose a certain amount of money to assign to the person. 
```

Comment:
originally i was going to make a suggestion to fix the phrasing of the part with the money but...
i know this kinda undoes the last suggestion i had, but reading the whole line over, i realize you need to just make it more clear In general, since since the money part is poorly written in general and doesn't even need to be separated from the person, in fact separating the parts of selecting a person and also choosing money makes it a lot harder to understand what yorue doing  as a result. can maybe change the entire line into something like
"each king must provide money and a teammate for tribute". you can decide the phrasing, but that's just an example of what you can do

Reply:
Alright I'll try